src/__tests__/Quote.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import NavBar from '../components/NavBar';
+import Quote from '../pages/Quote';
+import renderer from 'react-test-renderer';
+// const Quote = require('../pages/Quote');
+
+it('renders correctly', () => {
```

Comment:
- [ ] Kindly enable eslint in this module also.
- [ ] Also please remove the commented code.